proposals/nnnn-actors.md

```diff
@@ -120,21 +132,28 @@ Such types can then be used to annotate particular declarations that are isolate
 
 ```swift
 @UIActor
-func touchesEnded(_ touches: Set<UITouch>, with event: UIEvent?) {
+func touchesEnded(_ touches: Set<UITouch>, with event: UIEvent?) async {
```

Comment:
I actually don't know?

Was this meant to be synchronous?

Reply:
It's meant to be synchronous, which means it's caller must already be on the UI thread.

Reply:
Ah I see got it sorry :)